iamcristos/politico-app
# ft(endpoint): create politica party

#### What does this PR do?
endpoint that creates a new political party

#### Description of Task to be completed?
form inputed are passed, this input are first validated to ensure they are not emptied,
after validating a new political party is created

#### How should this be manually tested?
After cloning the reprository, run npm then test a post request on postman at '/api/v1/parties' ;
ensure to pass {name, hqaddress,logourl}. the id is generated automatically
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?

#163568731

#### Screenshots (if appropriate)
#### Questions:
